schemas/elf.yml

```diff
@@ -62,6 +62,43 @@
       level: extended
       example: Intel
 
+    - name: go_import_hash
+      short: A hash of the Go language imports in an ELF file.
+      description: >
+        A hash of the Go language imports in an ELF file. An import hash can be used to
```

Comment:
Is this a well-known hash standard? If so can we link to its documentation. If not, can we write a document somewhere the describes how this is calculated.

Reply:
The Go import hash is imphash/symhash but using the static Go imports rather than the dynamic imports. There is no standard. Where would you like a document put?

The parallels can be seen here https://github.com/elastic/toutoumomoma/blob/594ef30cb64046757baa988d4451b8f547dcb695/toutoumomoma.go#L149-L231

Reply:
I don't have a strong preference. How about in the https://github.com/elastic/toutoumomoma repo? You could point to the code as the reference implementation.

Reply:
Will do.

- [ ] Enhance readme in toutoumomoma.
- [ ] Pull description from `*File.ImportHash` out to godoc for both it and `*File.GoSymbolHash`
- [ ] Link to relevant locations in the ECS descriptions